docs/guide/authjs/session-data.md

```diff
@@ -87,13 +87,13 @@ const { data } = useAuth()
 
 When modifying the session or JWT objects, you may want to adjust the types accordingly, to ensure that you get proper intellisense and type support. [Module Augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation) can be used to inject additional type declarations into installed modules to overwrite or add additional data. 
 
-Begin by creating a new typescript file in the root of your project named: `next-auth.d.ts`. Typescript will automatically recognize that this file is augmenting the module types of `next-auth` (running under the hood).
+Begin by creating a new typescript file in the root of your project named: `auth.d.ts`. Typescript will automatically recognize that this file is augmenting the module types of `@auth/core` (running under the hood).
```

Comment:
We will need to revisit this to see what naming is needed exactly to modify the session data types! For now I went with `auth.d.ts`, however this may be incorrect!